extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -182,6 +185,13 @@ export class AmpA4A extends AMP.BaseElement {
   }
 
   /** @override */
+  buildCallback() {
+    const adType = this.element.getAttribute('type');
+    this.config = adConfig[adType];
+    user().assert(this.config, `Type "${adType}" is not supported in amp-ad`);
```

Comment:
Actually I would prefer separate a4a ad config from `_config.js`. 
I will revert this line of `user().assert` since `amp-ad` has already checked that the type is supported in A4A case.